### PR TITLE
Add pad_token_id to generate_kwargs.

### PR DESCRIPTION
**Pull request by oursland**
Eliminate the following warning by setting the `pad_token_id` to `generate_kwargs`.

> The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's attention_mask` to obtain reliable results.`

### CODE REVIEW

**Comment by thuwzy** [comment]
It looks strange to set `pad_token_id` as `tokenizer.eos_token_id`?

**Comment by oursland** [comment]
It did get rid of the message, but introduced another.  I will close this PR for now and re-open if I have a better solution.
